core/tests/test_utils.py

```diff
@@ -2462,21 +2523,34 @@ def urlfetch_mock(
             # Enables the testbed urlfetch mock.
             self.testbed.init_urlfetch_stub()
 
-    def count_jobs_in_taskqueue(self, queue_name):
+    def count_jobs_in_taskqueue(self, queue_name=None):
```

Comment:
Why do we need to keep the old method around? Why not replace all the existing functions with the new functionality? That would drastically reduce the number of files requiring changes.

Reply:
So are we still using both versions?